build.gradle

```diff
@@ -36,7 +36,8 @@ dependencies {
     implementation 'org.apache.camel.springboot:camel-jackson-starter:3.12.0'
     implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.13.1'
     implementation 'org.json:json:20211205'
-    implementation 'io.camunda:zeebe-client-java:1.3.1'
+    implementation 'io.camunda:zeebe-client-java:8.1.1'
+    implementation 'org.apache.camel:camel-http:3.12.0'
```

Comment:
why this line is showing in your change? is this dependency(org.apache.camel:camel-http) needed?

Reply:
@fynmanoj This is not a part of this change. This was added by mistake when resolving merge conflicts. This has been removed.